recipe/meta.yaml

```diff
@@ -15,28 +15,26 @@ build:
   # Build uses pep517 with a custom backend
   # console_scripts entrypoints aren't used, so this can't be noarch
   script:
-    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
     - mkdir -p $PREFIX/share/bash-completions/completions && echo complete -C aws_completer aws >$PREFIX/share/bash-completions/completions/aws  # [unix]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - python
     # The autocomplete index step is run on the build host
     # keep these in sync with host and run copies
-    - colorama >=0.2.5,<0.4.7                # [build_platform != target_platform]
-    - docutils >=0.10,<0.20                  # [build_platform != target_platform]
-    - cryptography >=40.0.0,<43.0.2          # [build_platform != target_platform]
-    - ruamel.yaml >=0.15.0,<=0.17.21         # [build_platform != target_platform]
-    - ruamel.yaml.clib >=0.2.0,<=0.2.8       # [build_platform != target_platform]
-    - prompt-toolkit >=3.0.24,<3.0.39        # [build_platform != target_platform]
-    - distro >=1.5.0,<1.9.0                  # [build_platform != target_platform]
-    - awscrt ==0.23.8                        # [build_platform != target_platform]
-    - python-dateutil >=2.1,<=2.9.0          # [build_platform != target_platform]
-    - jmespath >=0.7.1,<1.1.0                # [build_platform != target_platform]
-    - urllib3 >=1.25.4,<1.27                 # [build_platform != target_platform]
-    - zipp <3.21.0                           # [build_platform != target_platform]
-
+    - colorama >=0.2.5,<0.4.7
+    - docutils >=0.10,<0.20
+    - cryptography >=40.0.0,<43.0.2
+    - ruamel.yaml >=0.15.0,<=0.17.21
+    - ruamel.yaml.clib >=0.2.0,<=0.2.8
+    - prompt-toolkit >=3.0.24,<3.0.39
+    - distro >=1.5.0,<1.9.0
+    - awscrt ==0.23.8
+    - python-dateutil >=2.1,<=2.9.0
+    - jmespath >=0.7.1,<1.1.0
+    - urllib3 >=1.25.4,<1.27
+    - zipp <3.21.0
```

Comment:
```suggestion
```

I don't think these are necessary in the build section. They were only there for cross-compilation on conda-forge.